src/telepathy/Makefile.am

```diff
@@ -46,4 +48,4 @@ telepathy_sipe_LDADD = \
 	$(DBUS_GLIB_LIBS) \
 	$(GIO_LIBS) \
 	$(GOBJECT_LIBS) \
-	$(GLIB_LIBS)
+	$(GLIB_LIBS) -lpurple
```

Comment:
Okay, never mind. I'll anyway make changes to the build files before moving the code upstream. For now let's just keep telepathy disabled.